config/locales/simple_form.en.yml

```diff
@@ -82,6 +82,7 @@ en:
         mascot: Overrides the illustration in the advanced web interface.
         media_cache_retention_period: Downloaded media files will be deleted after the specified number of days when set to a positive value, and re-downloaded on demand.
         profile_directory: The profile directory lists all users who have opted-in to be discoverable.
+        publish_button_text: Overrides the text of the publish button.
```

Comment:
This should be in `/locales-glitch/simple_form.en.yml`.
Also the setting should have a title for localisation.